packages/@aws-cdk/aws-eks/lib/helm-release.ts

```diff
@@ -0,0 +1,86 @@
+import cfn = require('@aws-cdk/aws-cloudformation');
+import { Construct, Stack } from '@aws-cdk/core';
+import { Cluster } from './cluster';
+
+/**
+ * Helm Release Properties.
+ */
+export interface HelmReleaseProps {
+  /**
+   * The EKS cluster to apply this configuration to.
+   *
+   * [disable-awslint:ref-via-interface]
+   */
+  readonly cluster: Cluster;
+
+  /**
+   * The name of the release.
+   */
+  readonly name: string;
+
+  /**
+   * The name of the chart.
+   */
+  readonly chart: string;
+
+  /**
+   * The chart version to install.
+   * @default If this is not specified, the latest version is installed
+   */
+  readonly version?: string;
+
+  /**
+   * The repository.
+   * @default No repository will be used, which means that the chart needs to be an absolute URL.
+   */
+  readonly repository?: string;
+
+  /**
+   * The Kubernetes namespace scope of the requests.
+   * @default default
+   */
+  readonly namespace?: string;
+
+  /**
+   * The values.
```

Comment:
Provide a bit more details in the description and/or link a reference

Reply:
Added some more context to the description.